tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -0,0 +1,36 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Test for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use qesapdeployment;
+
+use constant GIT_CLONE_LOG => '/tmp/git_clone.log';
+
+sub run {
+    my ($self) = @_;
+    my $inventory = qesap_get_inventory(get_required_var('PUBLIC_CLOUD_PROVIDER'));
+
+    my $ansible_args = "-i $inventory -u cloudadmin -b --become-user=root";
+    assert_script_run('ansible all ' . $ansible_args . ' -a "pwd"');
```

Comment:
Pretty cool use of `ansible` to query things in all managed hosts; wonder if it would make sense to add an `ansible` or `qesap_ansible` function in `lib/qesapdeployment.pm` that deals with things like the ansible args for us to streamline the commands in the test modules. With such a function, code here can be:
```
use strict;
use warnings;
use Mojo::Base 'publiccloud::basetest';
use testapi;
use qesapdeployment;
use hacluster;

sub run {
    my ($sefl) = @_;
    ansible 'all', $_ for ('pwd', 'uname -a', 'cat /etc/os-release');
    ansible 'hana', 'ls -lai /hana/';
    ansible 'vmhana01', $_ for ('crm status', $crm_mon_cmd);
}
```
I'm assuming such a function will set appropriate args to `ansible`, get the inventory using `qesap_get_inventory` and get the value of the admin user from the qesap or terraform configuration directly.